vulnerability/fetcher_test.go

```diff
@@ -64,10 +67,19 @@ func TestVulnerabilityFetcher_FetchInstances(t *testing.T) {
 
 	assert.ElementsMatch(t, expectedInstances, receivedInstances)
 	wg.Wait()
+
+	// Verify no goroutines are leaking. Safest to keep this on top of the function.
+	// Go defers are implemented as a LIFO stack. This should be the last one to run.
+	goleak.VerifyNone(t, goleak.IgnoreCurrent())
```

Comment:
I don't think this invocation of goleak is correct here, with this syntax you are ignoring all goroutines. `VerifyNone` is supposed to be called with `defer` before you run any of the code you are testing as `IgnoreCurrent` will be initialized only with the goroutines that existed since before

Reply:
Yep, I think you are right, I will fix it